conformance/conformance_test.cc

```diff
@@ -756,23 +811,29 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
     {dbl(0.1), "0.1"},
     {dbl(1.7976931348623157e+308), "1.7976931348623157e+308"},
     {dbl(2.22507385850720138309e-308), "2.22507385850720138309e-308"}
-  });
+  }, true);
+  TestValidDataForType(FieldDescriptor::TYPE_DOUBLE, {
```

Comment:
It seems that TestValidDataForType() could test both proto2 and proto3 internally. Then you wouldn't need to change any of these call sites.